### PR TITLE
Feature/community store

### PR DESCRIPTION
**Pull request by J-Byron**
## Problem
*Document current behavior and why it's undesirable here.*

## Solution
*Document how the behavior was fixed or enhanced here.*

### Corresponding Branch
*Add link to crdschurch/repo-name-here#issue-number (if applicable). This helps the code reviewer know that corresponding work exists and where to find it.*

## Testing
*Include information on how to test your work here (if applicable).*

### CODE REVIEW

File: _assets/stylesheets/pages/_community-store.scss [hunk 1]
```diff
@@ -0,0 +1,150 @@
+#community-store-sub-nav {
+
+    width: 100%;
```
Comment (tcmacdonald) [inline _assets/stylesheets/pages/_community-store.scss:6]: Please correct the indentions to 2 spaces, per tab.

File: community-store.html [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+contentful_id: 63DCrL8G3LIBejQScAsjDm
+content_type: page
+title: Community Store
+permalink: "/community-store/"
+layout: container-fluid
+requires_auth: false
+search_excluded: false
+---
+
+<div class="row" style="max-width:100%; margin-left: 0;">
```
Comment (tcmacdonald) [inline community-store.html]: Can you please move all inline styles into your stylesheet?

File: community-store.html [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+        <a href="">
+          People
+        </a>
+      </span>
+  </div>
+</div>
+</div>
+
+<div class="jumbotron jumbotron-xl" id="community-store-intro"
+  style="background-image: url('https://crds-cms-uploads.imgix.net/content/images/streaming-anywhere-hero-off.jpg?auto=format,compress&w=597&fit=crop');">
```
Comment (tcmacdonald) [inline community-store.html]: Did you intend to crop this image to `597px` wide?

File: community-store.html [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+          consectetur adipiscing elit.
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container" style="padding: 0">
+  <div class="row card-container" style="margin: 30px auto 0px auto;">
+    <div class="card-deck carousel" data-crds-carousel="mobile-scroll" style="left: 30px;">
```
Comment (tcmacdonald) [inline community-store.html]: More inline styles here.

File: community-store.html [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+          </p>
+          <span class="font-size-small">
+            Some quick example text to build on the card title and make up the bulk of the card's content.
+          </span>
+        </div> -->
+        <div class="media">
+          <div class="media-left">
+            <a href="#">
+              <img class="media-object"
+                ix-src="//crds-cms-uploads.imgix.net/content/images/crossroads-go-header-imgs-nicaragua.jpg?w=108&h=108&fit=crop&format=auto,compress">
```
Comment (tcmacdonald) [inline community-store.html]: Can you make sure that all `` elements have `alt="..."` attributes?

File: _assets/stylesheets/pages/_community-store.scss [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+            font-size: 14px;
+            color: #ffffff;
+        }
+      }
+    }
+}
+
+#community-store-intro {
+
+    height: 597px;
```
Comment (tcmacdonald) [inline _assets/stylesheets/pages/_community-store.scss]: Can we make this a `min-height` declaration?

File: _assets/stylesheets/pages/_community-store.scss [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+    }
+    }
+}
+
+.card-container {
+    a {
+        color: $cr-cyan;
+    }
+
+    @media (min-width: 993px){
```
Comment (tcmacdonald) [inline _assets/stylesheets/pages/_community-store.scss]: Please use the variables provided by Bootstrap for the calculation stuff. `$screen-md` for example is `992px` so you could probably do this... 

```
@media (min-width: $screen-md + 1){ ... }
```

File: _assets/stylesheets/pages/_community-store.scss [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+    }
+}
+#groups-container {
+    width: 100%;
+
+    .heading-container{
+        margin-bottom: 45px;
+
+        span {
+            font-family: "lexia";
```
Comment (tcmacdonald) [inline _assets/stylesheets/pages/_community-store.scss]: Please use `$accent-font-face` for this font-family declaration. 

https://github.com/crdschurch/crds-styles/blob/development/assets/stylesheets/_variables.scss#L5

File: community-store.html [hunk 9]
```diff
@@ -0,0 +1,271 @@
...
+          ix-src="//crds-cms-uploads.imgix.net/content/images/crossroads-go-header-imgs-nicaragua.jpg"
+          alt="Card image caption">
+        <div class="group-cell-text">
+          <p class="font-family-condensed-extra text-uppercase">
+            Online Groups
+          </p>
+          <span class="font-size-small">
+            Some quick example text to build on the card title and make up the bulk of the card's content.
+          </span>
+        </div> -->
```
Comment (tcmacdonald) [inline community-store.html]: Please remove any commented out code.

**Review by tcmacdonald — CHANGES_REQUESTED** [review]
Looks good @J-Byron. I left some comments above ☝️. Please take a look and let me know if you have any questions. 

Thanks, Taylor

File: community-store.html [hunk 10]
```diff
@@ -0,0 +1,262 @@
...
+    </div>
+  </div>
+</div>
+
+<div class="jumbotron jumbotron-xl" id="community-store-intro"
+  style="background-image: url('https://crds-cms-uploads.imgix.net/content/images/streaming-anywhere-hero-off.jpg?auto=format,compress&fit=crop');">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-7" id="intro-text">
+        <h1 class="title text-uppercase">Get Involved</h1>
```
Comment (tcmacdonald) [inline community-store.html:49]: Can you reduce the weight on this element to `500`?

File: community-store.html [hunk 11]
```diff
@@ -0,0 +1,262 @@
...
+    <a class="btn btn-lg btn-secondary" href="#">View groups</a>
+  </div>
+</div>
+
+<div id="trips-container" class="jumbotron"
+  style="background-image: url('https://crds-cms-uploads.imgix.net/Uploads/community-store/community-store-trips-bg%403x.jpg?auto=format,compress&fit=crop');">
+  <img class="img-responsive imgix-fluid push-bottom mobile-flush-bottom" alt="" title=""
+    src="//crds-cms-uploads.imgix.net/content/images/crossroads-go-logo-white.png">
+
+  <h1 class="font-family-condensed-extra text-uppercase font-size-h1 flush-top">you were born for adventure</h1>
```
Comment (tcmacdonald) [inline community-store.html:244]: Can you convert this element to an `<h2>`? There should only be one `<h1>` element per page.

File: community-store.html [hunk 12]
```diff
@@ -0,0 +1,262 @@
...
+  <p>
+    And the opportunity is right in front of you.
+  </p>
+  <a class="btn btn-lg btn-secondary" href="#">View all trips</a>
+</div>
+
+<div id="people-container" class="jumbotron"
+  style="background-image: url('https://crds-cms-uploads.imgix.net/Uploads/community-store/finder-bg%403x.jpg?auto=format,compress&fit=crop');">
+  <h1 class="font-family-condensed-extra text-uppercase font-size-h1">Find people near you</h1>
```
Comment (tcmacdonald) [inline community-store.html:253]: Same as above, please make this an `<h2>`.

**Review by tcmacdonald — CHANGES_REQUESTED** [review]
Thanks @J-Byron – almost there!

I added a couple more inline comments ☝️ 

And lastly, we need to add some default background colors for any element that has a background image. Since those images are big, they may take a while to load and until they do, the white text isn't readable against a white background. Take a look at the following screenshot for some suggestions but use your best judgement if you think a different color works better... 

Community_Store___Crossroads

Let me know if you have any other questions. Thanks!

**Comment by seancdavis** [comment]
Closing this one to see if it resolves the duplicate commit issues here. I'll reopen if it does not.

**Pull request closed without merging**